client-src/elements/chromedash-app.ts

```diff
@@ -358,7 +358,8 @@ export class ChromedashApp extends LitElement {
       this.pageComponent.showQuery = false;
       this.pageComponent.rawQuery = parseRawQuery(ctx.querystring);
     });
-    page('/newfeatures', ctx => {
+    page('/newfeatures', () => page.redirect('/features'));
+    page('/features', ctx => {
```

Comment:
Question about this - I know the old /features page was not a part of SPA. Is that the reason why it is not routed here before? e.g. there was no `page('/features', ctx => {...`

Reply:
Yes, exactly.  And, now the old page is at `/oldfeatures` but that is not in the SPA routing because it is still an MPA page.